system-tests/tests/src/test/java/org/eclipse/dataspaceconnector/identityhub/systemtests/VerifiableCredentialsIntegrationTest.java

```diff
@@ -0,0 +1,104 @@
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.javafaker.Faker;
+import net.minidev.json.JSONObject;
+import org.eclipse.dataspaceconnector.iam.did.spi.credentials.CredentialsVerifier;
+import org.eclipse.dataspaceconnector.iam.did.spi.resolution.DidResolverRegistry;
+import org.eclipse.dataspaceconnector.identityhub.cli.IdentityHubCli;
+import org.eclipse.dataspaceconnector.identityhub.credentials.model.VerifiableCredential;
+import org.eclipse.dataspaceconnector.junit.extensions.EdcExtension;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import picocli.CommandLine;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.eclipse.dataspaceconnector.identityhub.credentials.VerifiableCredentialsJwtService.VERIFIABLE_CREDENTIALS_KEY;
+
+@IntegrationTest
+@ExtendWith(EdcExtension.class)
+class VerifiableCredentialsIntegrationTest {
+
+    static final Faker FAKER = new Faker();
+    static final String HUB_URL = "http://localhost:8182/api/identity-hub";
+    static final String AUTHORITY_DID = "did:web:localhost:authority";
+    static final String PARTICIPANT_DID = "did:web:localhost:participant";
+    static final String AUTHORITY_PRIVATE_KEY_PATH = "resources/jwt/authority/private-key.pem";
+    static final ObjectMapper MAPPER = new ObjectMapper();
+    static final VerifiableCredential VC1 = VerifiableCredential.Builder.newInstance()
+            .id(FAKER.internet().uuid())
+            .credentialSubject(Map.of(
+                    FAKER.internet().uuid(), FAKER.lorem().word(),
+                    FAKER.internet().uuid(), FAKER.lorem().word()))
+            .build();
+
+    CommandLine cmd = IdentityHubCli.getCommandLine();
+    StringWriter out = new StringWriter();
+    StringWriter err = new StringWriter();
+
+    @BeforeEach
+    void setUp(EdcExtension extension) {
+        cmd.setOut(new PrintWriter(out));
+        cmd.setErr(new PrintWriter(err));
+
+        extension.setConfiguration(Map.of(
+                "edc.identity.hub.url", HUB_URL,
+                "edc.iam.did.web.use.https", "false"));
+    }
+
+    @Test
+    void push_and_get_verifiable_credentials(CredentialsVerifier verifier, DidResolverRegistry resolverRegistry) throws Exception {
+        assertIdentityHubIsEmpty();
+        addVerifiableCredential();
+        assertGetVerifiableCredential(verifier, resolverRegistry);
+    }
+
+    private void assertIdentityHubIsEmpty() throws JsonProcessingException {
+        int result = cmd.execute("-s", HUB_URL, "vc", "list");
+        assertThat(result).isEqualTo(0);
+
+        var claims = MAPPER.readValue(out.toString(), new TypeReference<List<Map<String, Object>>>() {});
+        assertThat(claims).describedAs("Identity Hub already contains Verifiable Credentials").isEmpty();
```

Comment:
```suggestion
        assertThat(claims).as("Identity Hub contains no Verifiable Credentials").isEmpty();
```
Note that `describedAs` is an alias for `as` (which can be used for fluency when useful), and should describe the assertion, not the error message on violation